config/action.d/firewallcmd-allports.conf

```diff
@@ -29,10 +25,9 @@ actionunban = firewall-cmd --direct --remove-rule ipv4 filter f2b-<name> 0 -s <i
 
 [Init]
 
-# Default name of the chain
-#
 name = default
```

Comment:
I'll fix that. I don't know why I got the input chain mixed up with the f2b chains.